src/libcore/fmt/mod.rs

```diff
@@ -791,23 +883,33 @@ impl Show for () {
 
 impl<T: Copy + Show> Show for Cell<T> {
     fn fmt(&self, f: &mut Formatter) -> Result {
-        write!(f, "Cell {{ value: {} }}", self.get())
+        write!(f, "Cell {{value: {:?}}}", self.get())
```

Comment:
Or rather, does `derive` not stick spaces around it? This should stick to whatever `derive` does at least.